docs/src/modules/components/TopLayoutBlog.js

```diff
@@ -249,7 +298,8 @@ function TopLayoutBlog(props) {
           </Link>
           {headers.title ? (
             <React.Fragment>
-              <time dateTime={headers.date} className={classes.time}>
+              {/* eslint-disable-next-line react/no-unknown-property */}
+              <time datetime={headers.date} className={classes.time}>
```

Comment:
It renders as ` <time dateTime>` in the SSR output, while it should be lowercase https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time.